config/routes.rb

```diff
@@ -1,13 +1,9 @@
 Rails.application.routes.draw do
   root to: 'start_page#show'
 
-  get 'personal_details', to: 'personal_details#new'
-  post 'personal_details', to: 'personal_details#create'
-  patch 'personal_details', to: 'personal_details#update'
+  resources :personal_details, only: %i[new create edit update]
```

Comment:
`resources` does add ID parameters to the URLs, which feels a bit exposed — we agree there's a benefit to using standard Rails routing here, though

Reply:
Since we'll need to allow more candidates to edit their details,  we would eventually need an ID to identify them. 
Regarding `feels a bit exposed`, we'll eventually add authentication/authorization logic.